doc/build-netbsd.md

```diff
@@ -1,81 +1,145 @@
+ [boost](https://pkgsrc.se/meta-pkgs/boost)       | Utility    | Library for threading, data structures, etc
+ [libevent](https://pkgsrc.se/devel/libevent)     | Networking | OS independent asynchronous networking
+
+
+The following dependencies are **optional**:
+
+  Library                                            | Purpose          | Description
+  ---------------------------------------------------|------------------|----------------------
+  [db4](https://pkgsrc.se/databases/db4)             | Berkeley DB      | Wallet storage (only needed when wallet enabled)
+  [qt5](https://pkgsrc.se/x11/qt5)                   | GUI              | GUI toolkit (only needed when GUI enabled)
+  [qrencode](https://pkgsrc.se/converters/qrencode)  | QR codes in GUI  | Generating QR codes (only needed when GUI enabled)
+  [zeromq](https://pkgsrc.se/net/zeromq)             | ZMQ notification | Allows generating ZMQ notifications (requires ZMQ version >= 4.0.0)
+  [sqlite3](https://pkgsrc.se/databases/sqlite3)     | SQLite DB        | Wallet storage (only needed when wallet enabled)
+  [python37](https://pkgsrc.se/lang/python37)        | Testing          | Python Interpreter (only needed when running the test suite)
+
+  See [dependencies.md](dependencies.md) for a complete overview.
+
+
+## Preparation
+
+### 1. Install Required Dependencies
+Install the required dependencies the usual way you [install software on NetBSD](https://www.netbsd.org/docs/guide/en/chap-boot.html#chap-boot-pkgsrc) - either with `pkg_add` or `pkgin`. The example commands below use `pkgin` which is usually run as `root` or via `sudo`.
+
+```bash
+pkgin install autoconf automake libtool pkg-config git gmake boost libevent
 
 ```
-autoconf
-automake
-boost
-git
-gmake
-libevent
-libtool
-pkg-config
-python37
 
+See [dependencies.md](dependencies.md) for a complete overview.
+
+### 2. Clone Bitcoin Repo
+Now that `git` and all the required dependencies are installed, let's clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+``` bash
 git clone https://github.com/bitcoin/bitcoin.git
 ```
 
-See [dependencies.md](dependencies.md) for a complete overview.
+### 3. Install Optional Dependencies
+
+#### Wallet Dependencies
+It is not necessary to build wallet functionality to run bitcoind or the GUI. To enable legacy wallets, you must install `db4`. To enable [descriptor wallets](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md), `sqlite3` is required. Skip `db4` if you intend to *exclusively* use descriptor wallets
+
+###### Legacy Wallet Support
+`db4` is required to enable support for legacy wallets. Skip if you don't intend to use legacy wallets
+
+```bash
+pkgin install db4
+```
 
-### Building BerkeleyDB
+###### Descriptor Wallet Support
+
+`sqlite3` is required to enable support for descriptor wallets. Skip if you don't intend to use descriptor wallets.
+``` bash
```

Comment:
When adding code blocks - add a newline before the ticks - it is a markdown thing...
This ensures better cross platform rendering...

![Screen Shot 2020-12-13 at 11 38 28 PM](https://user-images.githubusercontent.com/152159/102041016-58dcf400-3d9c-11eb-8e99-3030988ed785.png)


###### Descriptor Wallet Support

`sqlite3` is required to enable support for descriptor wallets. Skip if you don't intend to use descriptor wallets.

``` bash
pkgin install sqlite3
```
---

Reply:
addressed in 1e88f35883ac34914b1a1177244ebd7ff4977ef4